pkgs/top-level/all-packages.nix

```diff
@@ -10892,8 +10892,6 @@ with pkgs;
 
   phodav = callPackage ../tools/networking/phodav { };
 
-  phodav_2_0 = callPackage ../tools/networking/phodav/2.0.nix { };
```

Comment:
wouldn't we also want to remove this file?

Reply:
Oh, thanks!